syntax = "proto3";

service Id_Manager {
    rpc getId (req) returns (res) {}
    rpc setId (req) returns (res) {}
}

message req {
    string id = 1;
}

message res {
    string id = 1;
}